charts/datadog/templates/agent-clusterchecks-deployment.yaml

```diff
@@ -211,6 +211,15 @@ spec:
           - name: config
             mountPath: {{ template "datadog.confPath" . }}
             readOnly: false # Need RW for config path
+{{- if eq (include "should-mount-fips-configmap" .) "true"}}            
+          - name: fips-config
+            mountPath: /etc/datadog-fips-proxy
+            readOnly: false # Need RW to mount to config path
+          - name: fips-proxy-cfg
+            mountPath: /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
```

Comment:
Do we need to worry about UIDs to be able to read the mount? [We don't run the fips-proxy container process as root](https://github.com/DataDog/fips-proxy/blob/main/Dockerfiles/amd64/Dockerfile#L110) so hopefully it won't be an issue.

Reply:
I'm not too sure what you mean. Do you mean whether the fips-proxy container process would be able to read the config file from the mount if the mount is read-only?

Reply:
The concern is mainly around the fips-proxy being able to read that file since UID/GID of fips-proxy process won't be root. It's more just a question if that will be a problem for the mount or not.

Reply:
Got it. I tested the changes manually in EKS and the `fips-proxy` container in the cluster-agent pod log many lines of output indicating that the FIPS mode is enabled and servers are `UP/READY`. I'm not sure if I should include redacted log lines so I'll leave them out.